docker/nexus/Dockerfile

```
@@ -0,0 +1,43 @@
+FROM python:3.10-slim AS builder
```

using syntax is considered good practice.

```suggestion
#syntax=docker/dockerfile:1

FROM python:3.10-slim AS builder
```